ndcube/global_coords.py

```diff
@@ -11,12 +11,12 @@
 
 from ndcube.utils.wcs import validate_physical_types
 
-__all__ = ['GlobalCoordsABC', 'GlobalCoords']
+__all__ = []
```

Comment:
I have done this in a few places and well, my idea is that since this is actually in the main namespace, we want to avoid this all and it causing problems with multiple places a user might import it from.